### PR TITLE
Disclosure pagination

### PR DESCRIPTION
**Pull request by enzoampil**
@jpdeleon Multi page disclosure querying should work now, by identifying the number of pages in the results and looping through each of the pages.

Sampe code (2 pages):

```
DATE_START = "01-01-2018"
DATE_END = "12-01-2019"
cd = DisclosuresPSE(
    symbol=PHISIX_SYMBOL, start_date=DATE_START, end_date=DATE_END
)
cd.company_disclosures.shape
# (59, 7)
```

### CODE REVIEW

File: fastquant/disclosures.py [hunk 1]
```diff
@@ -195,6 +206,18 @@ def get_company_disclosures(self):
         html = response.text
         # Indicating the parser (e.g.  lxml) removes the bs warning
         parsed_html = BeautifulSoup(html, "lxml")
+        current_page, page_count, results_count = re.findall(
+            "[^A-Za-z\[\]\/\s]+",
+            parsed_html.find("span", {"class": "count"}).text,
+        )
+        current_page, self.page_count, self.results_count = (
```
Comment (jpdeleon) [inline fastquant/disclosures.py:213]: Is it OK to change data type of page_count from bool to int here?

File: fastquant/disclosures.py [hunk 2]
```diff
@@ -73,10 +75,13 @@ def __init__(
             start date with format %m-%d-%Y
         end_date : str
             end date with format %m-%d-%Y
+        start_page : int
+            first disclosure page to start with
         """
         self.symbol = symbol.upper()
         self.start_date = start_date
         self.end_date = TODAY if end_date is None else end_date
+        self.start_page = start_page
```
Comment (jpdeleon) [inline fastquant/disclosures.py:84]: This always starts at page 1, isn't it? Is there a case otherwise?

File: fastquant/disclosures.py [hunk 3]
```diff
@@ -144,14 +150,18 @@ def get_stock_data(self):
...
         """
+        Gets company disclosures for one page
+
         symbol : str
             Ticker of the pse stock of choice
         start_date date : str (%m-%d-%Y)
             Beginning date of the disclosure data pull
         end_date : str (%m-%d-%Y)
             Ending date of the disclosure data pull
+        page_count : bool
```
Comment (jpdeleon) [inline fastquant/disclosures.py:163]: minor: should be page not page_count

File: fastquant/disclosures.py (hunk 2 above)
Comment (enzoampil) [reply fastquant/disclosures.py:84]: Actually good point. Will just set it to 1

**Pull request merged**